src/toxcore/dht/server.rs

```diff
@@ -123,14 +128,24 @@ impl Server {
             None
         }
     }
+    /// remove timed-out clients
+    pub fn remove_timedout_client(&self) -> IoFuture<()> {
+        let mut state = self.state.write().expect("Failed to obtain ServerState lock");
+        state.peers_cache.retain(|&_pk, ref client|
+            client.last_resp_time.elapsed() <= Duration::from_secs(122));
```

Comment:
Make a const please instead of magic number.

Reply:
Or better make it argument for function